app/scripts/modules/core/src/managed/overview/artifact/VersionMetadata.tsx

```diff
@@ -0,0 +1,56 @@
+import { DateTime } from 'luxon';
+import React from 'react';
+
+import { IconTooltip } from 'core/index';
+import { RelativeTimestamp } from '../../RelativeTimestamp';
+
+export const VersionMetadata = ({
+  buildNumber,
+  author,
+  deployedAt,
+  buildDuration,
+  commitsBehind,
+}: {
+  buildNumber?: string;
+  author?: string;
+  deployedAt?: string;
+  buildDuration?: string;
+  commitsBehind?: number;
+}) => {
+  const elements = [
```

Comment:
I dislike reading this code, but I see why you did it this way.

Reply:
Yeah, it's ugly. Do you have a better idea?

Reply:
Since `buildNumber` always exists, can you couple the separator to the data, and just directly render all the elements? Ex: 

```js
<div className="version-metadata">
    <>Build #{buildNumber}</>
    {author && <span>•</span>}
    {author && <AuthorElement />}
    {deployedAt && <span>•</span>}
    {deployedAt && <DeployAtElement />}
    {buildDuration && <span>•</span>}
    {buildDuration && <BuildDurationElement />}
    {commitsBehind && <span>•</span>}
    {commitsBehind && <CommitsBehindElement />}     
 </div>
```

Reply:
I only want to add the separator if an element exists on both sides

Reply:
I have an idea :)